watchtower/blob/justice_kit_test.go

```diff
@@ -38,14 +41,22 @@ func makeAddr(size int) []byte {
 	return addr
 }
 
+func makeSchnorrSig(i int) lnwire.Sig {
+	var sigBytes [64]byte
+	binary.BigEndian.PutUint64(sigBytes[:8], uint64(i))
+
+	sig, _ := lnwire.NewSigFromSchnorrRawSignature(sigBytes[:])
+
+	return sig
+}
+
 type descriptorTest struct {
 	name                 string
 	encVersion           Type
 	decVersion           Type
 	sweepAddr            []byte
 	revPubKey            *btcec.PublicKey
 	delayPubKey          *btcec.PublicKey
-	csvDelay             uint32
```

Comment:
Rationale for removal? Also generally in PR could use more commit body messages all throughout.